api/pages/saleshomepage.html

```diff
@@ -172,17 +171,47 @@ <h6>Recipients:</h6>
         $('#sidebarCollapse').on('click', function () {
             $('#sidebar').toggleClass('active')
         });
-    });
-</script>
 
 
+         $.getJSON("../message/readforSM.php", function (data) {
+            
+            var messagestrTD = '<ul class="media-list">';
+
+            $.each(data.records, function(key,val){
+            var message = val.content;
+            var date_time = val.date_created;
+            var role =   val.role;
+      
+            messagestrTD += `<li class="media">   
+                            <div class="media-body">
+                            <strong class="text-primary">`+ role +`</strong>
+                            <small class="text-muted">` + date_time + `</small>
+                            <p>` + message + `</p>
+                        </div></li>`;
+        
+           });
+
+           $('#announcementsSM').append(messagestrTD);
+       });
+	
+
+
+
+ });
+</script>
+
+<script>
+ $(document).on("click", "#modal-button", function(event){
+    alert( $("#messageContent")[0].value ); 
+});
+</script>
 
 </body>
 
 	<script type="text/javascript">
 		function startup(){
 			loggedin();
-			checkPermissions("Sales Manager");
+			checkPermissions("Admin");
```

Comment:
nope, checking for Sales Manager will handle it correctly
we need to know if the user is a Sales Manager versus an admin because if they aren't an admin, the button for a new message will get hidden (which I'll add)